aten/src/ATen/SparseTensorImpl.h

```diff
@@ -80,6 +81,27 @@ struct SparseTensorImpl : public TensorImpl {
     }
     sparseDims_ = sparseDims;
     denseDims_ = denseDims;
+    #else
+    if (size.size() == 0) {
+      // An empty sparse tensor should be a 1-dimensional tensor of size [0],
```

Comment:
`size.size() == 0` is usually a scalar (0 dimensional) -- why is it different here?